uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java

```diff
@@ -54,7 +54,7 @@
 public class InfinispanTestSuite {
 
     @ClassRule
-    public static ContainerDslRule infinispan = new ContainerDslRule("jboss/infinispan-server:9.3.2.Final")
+    public static ContainerDslRule infinispan = new ContainerDslRule("jboss/infinispan-server:9.4.9.Final")
```

Comment:
could we have this filtered by maven from https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/841856b8a8de72586d718eae2bd8362fd91adaad/pom.xml#L243 version property?

Reply:
+1

Reply:
@mareknovotny but is this going to work if someone wants to run this test alone without maven, or from IDE?

Reply:
@adrielparedes it won't work without Maven or support of that in IDE. I know m2eclipse does that for Eclipse and filtering worked for me.
I am not sure about Intellij Idea, but I guess it supports it in some way too even you can run a test through Maven goal execution I think.

Reply:
if for any reason, the IDE doesn't work @adrielparedes, I'd prefer to have someone touching it manually local to make it work.. instead to have the version fixed in the code.

+1 for @mareknovotny

Reply:
@porcelli @mareknovotny great I will update the PR today. Thanks guys